tests/e2e/e2e_exec_test.go

```diff
@@ -11,10 +11,92 @@ import (
 	"cosmossdk.io/math"
 	"github.com/cosmos/cosmos-sdk/client/flags"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 	"github.com/ory/dockertest/v3/docker"
 )
 
-func (s *IntegrationTestSuite) execBankSend(c *chain, valIdx int, from, to, amt, fees string, expectErr bool) {
+const (
+	flagSpendLimit      = "spend-limit"
+	flagAllowedMessages = "allowed-messages"
+	flagFrom            = "from"
+	flagHome            = "home"
+	flagFees            = "fees"
+	flagFeeGranter      = "fee-granter"
+	flagGas             = "gas"
+	flagOutput          = "output"
+	flagChainID         = "chain-id"
+	flagBroadcastMode   = "broadcast-mode"
+	flagKeyringBackend  = "keyring-backend"
+)
+
+type flagOption func(map[string]interface{})
+
+// withKeyValue add a new flag to command
```

Comment:
// withKeyValue add a new flag to command or change keyValue of an existing flag ?